# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/generic/nsGfxScrollFrame.cpp
# Commit: a2da895a58ce
# Full Hash: a2da895a58ce094d55e6a2e040dd648962b39868
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2022-02-22 09:37:09
# Regressor Bug: 1714584
# File Overlap Count: 2
# Description:
#   Bug 1714584 - Part 1: Decouple nsDisplayList internal list from nsDisplayItems r=mstange
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138152
# ==============================================================================

diff -r c3b3315d2344 -r a2da895a58ce layout/generic/nsGfxScrollFrame.cpp
--- a/layout/generic/nsGfxScrollFrame.cpp	Mon Feb 21 21:57:03 2022 +0000
+++ b/layout/generic/nsGfxScrollFrame.cpp	Tue Feb 22 00:44:25 2022 +0000
@@ -4132,7 +4132,7 @@
     // Effectively we are double clipping to the viewport, at potentially
     // different async scales.
 
-    nsDisplayList resultList;
+    nsDisplayList resultList(aBuilder);
     set.SerializeWithCorrectZOrder(&resultList, mOuter->GetContent());
 
     if (blendCapture.CaptureContainsBlendMode()) {